channels/de.m3u

```diff
@@ -15,8 +15,6 @@ https://alex-stream.rosebud-media.de/live/alexlivetv40.smil/playlist.m3u8
 https://stream01.stream.welocal.world/stream/alg/ngrp:alg.stream_all/playlist.m3u8
 #EXTINF:-1 tvg-id="AnixeHD.de" tvg-name="Anixe HD" tvg-country="DE" tvg-language="German" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c0/Anixe_HD.svg/641px-Anixe_HD.svg.png" group-title="",Anixe HD (720p)
 https://ma.anixa.tv/clips/stream/anixehd/index.m3u8
-#EXTINF:-1 tvg-id="AntenneVorarlbergWebradio.de" tvg-name="Antenne Vorarlberg (Webradio)" tvg-country="DE" tvg-language="" tvg-logo="https://i.imgur.com/TpBbZwq.png" group-title="",Antenne Vorarlberg (Webradio) (720p)
-http://5857db5306b83.streamlock.net/antennevorarlberg-live/_definst_/mp4:livestream/playlist.m3u8
```

Comment:
Works for me.

Reply:
I've already marked the link as "Not 24/7"